.github/container/Dockerfile.base

```diff
@@ -50,3 +50,9 @@ ENV PATH=/opt/amazon/efa/bin:${PATH}
 ###############################################################################
 
 RUN ln -s /opt/nvidia/nsight-compute/*/host/target-linux-x64/nsys /usr/local/cuda/bin
+
+###############################################################################
+## Add the systemcheck to the entrypoint.
+###############################################################################
+
+COPY systemchecks.sh /opt/nvidia/entrypoint.d/jax-systemchecks.sh
```

Comment:
Can we use the same file name here? Something like
```
COPY system-checks.sh /opt/nvidia/entrypoint.d/
```

Reply:
Yes I can make that change. Just worried about someone accidentally overwriting that name downstream.